Makefile

```diff
@@ -0,0 +1,32 @@
+# Project Setup
+PROJECT_NAME := platform-ref-aws
+PROJECT_REPO := github.com/upbound/$(PROJECT_NAME)
+
+PLATFORMS ?= linux_amd64 linux_arm64
+include build/makelib/common.mk
+
+# ====================================================================================
+# Setup Kubernetes tools
+
+UP_VERSION = v0.13.0
+UP_CHANNEL = stable
+
+-include build/makelib/k8s_tools.mk
+# ====================================================================================
+# Setup XPKG
+XPKG_REG_ORGS ?= xpkg.upbound.io/upbound
+# NOTE(hasheddan): skip promoting on xpkg.upbound.io as channel tags are
+# inferred.
+XPKG_REG_ORGS_NO_PROMOTE ?= xpkg.upbound.io/upbound
+XPKGS = $(PROJECT_NAME)
+-include build/makelib/xpkg.mk
+
+CROSSPLANE_NAMESPACE = upbound-system
+-include build/makelib/local.xpkg.mk
+
+# ====================================================================================
+# Targets
+
+# We must ensure up is installed in tool cache prior to build as including the k8s_tools machinery prior to the xpkg
+# machinery sets UP to point to tool cache.
+build.init: $(UP)
```

Comment:
can we add `submodules` target to the Makefile to enable the local `build` invocation?

Reply:
Done!